src/dune/workspace.mli

```diff
@@ -76,7 +76,12 @@ val to_dyn : t -> Dyn.t
 val hash : t -> int
 
 val init :
-  ?x:Context_name.t -> ?profile:Profile.t -> ?path:Path.t -> unit -> unit
+     ?x:Context_name.t
+  -> ?profile:Profile.t
+  -> ?instrument_with:string list
```

Comment:
Why not use `Lib_name.t` here? We are using typed representations for the other arguments.

Reply:
Fixed, thanks.